api/src/pcapi/core/educational/api/offer.py

```diff
@@ -11,14 +11,17 @@
 from pcapi.core.educational import exceptions
 from pcapi.core.educational import models as educational_models
 from pcapi.core.educational import repository as educational_repository
-from pcapi.core.educational import validation
+from pcapi.core.educational import validation as educational_validation
 from pcapi.core.educational.adage_backends.serialize import serialize_collective_offer
 from pcapi.core.educational.exceptions import AdageException
 from pcapi.core.educational.models import HasImageMixin
 from pcapi.core.offerers import api as offerers_api
 from pcapi.core.offerers import exceptions as offerers_exceptions
 from pcapi.core.offerers import models as offerers_models
+from pcapi.core.offerers import repository
+from pcapi.core.offers import exceptions as offers_exception
 from pcapi.core.offers import validation as offer_validation
+from pcapi.core.offers.models import OfferValidationStatus
 from pcapi.core.users.models import User
```

Comment:
```suggestion
from pcapi.core.offers. import models as offers_models
```
et revoir la ou User et OfferValidationStatus sont utilisés